docs/en/ingest-management/commands.asciidoc

```diff
@@ -834,6 +835,63 @@ elastic-agent install --url=https://fleet-server:8220 \
   --certificate-authorities=/path/to/ca.crt
 ----
 
+++++
+<hr>
+++++
+
+[discrete]
+[[elastic-agent-otel-command]]
+== elastic-agent otel
+
+preview::[]
+
+Run {agent} as an <<otel-agent,OpenTelemetry Collector>>. 
+
+[discrete]
+=== Synopsis
+
+[source,shell]
+----
+elastic-agent otel [flags]
+elastic-agent otel [command]
+----
+
+[discrete]
+=== Available commands
+
+`--validate`::
+Validates the OpenTelemetry collector configuration without running the collector.
+
+[discrete]
+=== Flags
+
+`--config --config=file:/path/to/first --config=file:path/to/second`::
```

Comment:
Using `--config` flag without a value is incorrect. Again, maybe I'm misinterpreting the markup?

```suggestion
`--config=file:/path/to/first --config=file:path/to/second`::
```

Reply:
Fixed! 
![Screenshot 2024-06-18 at 1 27 37 PM](https://github.com/elastic/ingest-docs/assets/41695641/f07a8105-4821-4d6b-ad91-1041afd34ab7)